src/Domain/CoordinateMaps/Equiangular.hpp

```diff
@@ -0,0 +1,106 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines the class Equiangular.
+
+#pragma once
+
+#include <array>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+
+namespace PUP {
+class er;
+}  // namespace PUP
+
+namespace CoordinateMaps {
+
+/*!
+ * \ingroup CoordinateMapsGroup
+ * \brief Non-linear map from \f$\xi \in [A, B]\rightarrow x \in [a, b]\f$.
+ *
+ * The formula for the mapping is:
+ * \f[
+ * x = \frac{a}{2} (1-\mathrm{tan}(\frac{\pi(2\xi-B-A)}{4(B-A)}) +
```

Comment:
make sure these are aligned at the equal sign (might have to use the align env) again for below equation